### ISSUE TITLE
[BUG] --timeout flag to cmd run does not work for values above 10 seconds

### ISSUE THREAD

**Issue by aniaptebsft** [gogrlx/grlx#108]
**Please describe the issue.**
The --timeout flag to cmd run appears to take a max value of 10 seconds. Giving a value of say 15 seconds has no effect - the command still times out after 10 seconds. However it does work if the value is given less than 10, say 8 seconds.

**Version Information**
```
$ grlx version
CLI Version:
        Tag: v1.0.0
        Commit: c8745ea
        Arch: linux
        Compiler: go1.21.3
Farmer Version:
        Tag: v1.0.0
        Commit: c8745ea
        Arch: linux
        Compiler: go1.21.3
```

**Does this bug exist on the tip of master?**
Yes

**Evidence**
```
$ time -p grlx --target "*" cmd run --timeout 99  -- sleep 15
2024/03/07 12:19:56 Post "https://grlx-server-1.internal.vpc:5405/cmd/run": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
panic: Post "https://grlx-server-1.internal.vpc:5405/cmd/run": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

goroutine 1 [running]:
log.Panic({0xc000079a98?, 0x5?, 0xc000146400?})
        /usr/lib/go/src/log/log.go:432 +0x5a
github.com/gogrlx/grlx/cmd/grlx/cmd.glob..func6(0xc00011a100?, {0xc0001463f0?, 0x4?, 0xdf8b75?})
        /home/tai/code/foss/grlx/cmd/grlx/cmd/cmd.go:67 +0x35b
github.com/spf13/cobra.(*Command).execute(0x1533d20, {0xc000146380, 0x7, 0x7})
        /home/tai/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xaa3
github.com/spf13/cobra.(*Command).ExecuteC(0x1535140)
        /home/tai/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
        /home/tai/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/gogrlx/grlx/cmd/grlx/cmd.Execute({{0xdf93d1, 0x5}, {0xf66c10, 0x8}, {0xf66c00, 0x7}, {0xf665f8, 0x6}})
        /home/tai/code/foss/grlx/cmd/grlx/cmd/root.go:37 +0x97
main.main()
        /home/tai/code/foss/grlx/cmd/grlx/main.go:25 +0xce
real 10.01
user 0.00
sys 0.00
```

**Comment by aniaptebsft** [comment]
I suspect its because of this: https://github.com/gogrlx/grlx/blob/c7d1ee16cc38721585e4b751f9bdd08679171ab5/api/client/transport.go#L54

The go http client timeout is hardcoded to 10 seconds, so the low level http client timesout after 10 seconds.

**Comment by taigrr** [comment]
Thank you for the bug report, and also for doing a bit of digging to determine the potential cause. 


This worked in the past, so it's possible a go compiler version or module update of the stdlib broke somewhere along the way. I'll have to add a test case for ~15 seconds in the future alongside the fix to prevent future regressions.

I'll be able to start addressing this within the week. There was a related bug as well, which I haven't publicized just yet, regarding package upgrades. 


Thank you!

**Comment by taigrr** [comment]
Closed in [#0e14d7](https://github.com/gogrlx/grlx/commit/a92fcea9cf24dc10b0cac77c55700d020f620614)

Released in v1.0.3
